### PR TITLE
Dynamic case tiles

### PR DESCRIPTION
**Pull request by dcluna**
Solving http://manage.dimagi.com/default.asp?179028.

### CODE REVIEW

**Comment by phillipm** [comment]
Do you have an app I can install to test out these changes?

**Comment by dcluna** [comment]
https://www.commcarehq.org/a/rec/apps/view/f7036b646c12ba5acb9977fef748088a/releases/
, child visit

File: app/src/org/commcare/dalvik/activities/EntitySelectActivity.java [hunk 1]
```diff
@@ -884,6 +892,35 @@ public void deliverResult(List<Entity<TreeReference>> entities, List<TreeReferen
         this.startTimer();
     }
 
+    private void setupDivider(ListView view) {
+        int viewWidth = view.getWidth();
+        float density = getResources().getDisplayMetrics().density;
+        int viewWidthDP = (int)(viewWidth / density);
```
Comment (phillipm) [inline app/src/org/commcare/dalvik/activities/EntitySelectActivity.java]: ```viewWidthDP``` doesn't seem to be used, other than for debug logging.

File: app/src/org/commcare/dalvik/activities/EntitySelectActivity.java [hunk 2]
```diff
@@ -884,6 +892,35 @@ public void deliverResult(List<Entity<TreeReference>> entities, List<TreeReferen
         this.startTimer();
     }
 
+    private void setupDivider(ListView view) {
+        int viewWidth = view.getWidth();
+        float density = getResources().getDisplayMetrics().density;
+        int viewWidthDP = (int)(viewWidth / density);
+        // sometimes viewWidth is 0, and in this case we default to a reasonable value taken from dimens.xml
+        int dividerWidth = viewWidth == 0 ? (int)getResources().getDimension(R.dimen.entity_select_divider_left_inset) : (int)(0.15 * viewWidth);
```
Comment (phillipm) [inline app/src/org/commcare/dalvik/activities/EntitySelectActivity.java]: What does the 0.15 signify here?

Comment (phillipm) [inline app/src/org/commcare/dalvik/activities/EntitySelectActivity.java]: Oh nevermind, it is the percentage of the view's width dedicated to the divider

**Comment by phillipm** [comment]
Tried running the REC app on this branch and on master and still wasn't able to figure out what is being fixed here.

**Pull request merged**